docs/editor/voice.md

```diff
@@ -0,0 +1,59 @@
+
+## Editor dictation
+
+You can dictate into the editor via these commands: **Voice: Start Dictation in Editor** (`kb(workbench.action.editorDictation.start)`) and **Voice: Stop Dictation in Editor** (`kb(workbench.action.editorDictation.stop)`). Once started, a little microphone icon appears where the cursor is, awaiting your voice input:
+
+![Editor Dictation Mode](images/accessibility/editor-dictate.png)
+
+You can press and hold the keybinding for the voice start command (`kb(workbench.action.editorDictation.start)`) to enable **walky-talky mode**. Voice recognition is active until you release the keys, after which the request is submitted automatically.
+
+**Note:** dictation even works in other places where a rich editor is used, such as the SCM commit input box and the comments input field when reviewing pull requests.
+
+## Voice in GitHub Copilot Chat
+
+You can use your voice to talk to [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) easily. The command **Voice: Start Voice Chat** (`kb(workbench.action.chat.startVoiceChat)`) brings up a voice chat, regardless of where the current focus is. If the focus is in the editor, inline chat is started, otherwise the Chat view is opened.
+
+![Screenshot of the chat input field showing an active microphone icon to enter a voice chat message](images/accessibility/voice-chat.png)
+
+**Note:** when using voice for GitHub Copilot Chat, the chat prompt will automatically submit when you pause. You can configure the time to wait before submitting via the `accessibility.voice.speechTimeout` setting, or disable this functionality when you configure the setting to `0`.
+
+## Walky talky mode
+
+When using keyboard shortcuts to start voice (`kb(workbench.action.editorDictation.start)` or `kb(workbench.action.chat.startVoiceChat)`), either in the editor or chat, you can press and hold the keybinding to start the voice recognition. When you release the keybinding, voice recognition will stop. In addition, when used in chat, the prompt will be submitted.
+
+## "Hey Code"
+
+It is possible to enable a mode where VS Code will always listen for the phrase "Hey Code" to start a voice chat session. Configure the `accessibility.voice.keywordActivation` setting accordingly to enable this. When VS Code is listening for "Hey Code", a microphone icon appears in the status bar to indicate as such:
+
+![Screenshot of a status bar entry to signal active listening to "Hey Code"](images/accessibility/hey-code.png)
+
+## Support for multiple languages
+
+You can select from one of the 26 supported languages by using the `accessibility.voice.speechLanguage` setting.
+
+Each language for the speech extension comes as its own extension. When you start speech recognition for the first time, you will see an extension installation for each language you selected.
+
+## Next steps
```

Comment:
Should we have a call to action to ask for feedback?
I feel like this feature is still pretty new and we could mention that any feedback is welcome?

Maybe not needed. Not sure.

Reply:
Yeah, though thats probably true for a lot of docs pages. Should there be a generic button in the docs to report issues or feedback? I think that would be cool.

Reply:
We have feedback buttons in the right sidebar to either ask for features or report issues :)
![image](https://github.com/microsoft/vscode-docs/assets/1908215/5ae84d55-b210-463e-83b5-05e7038ae050)

Reply:
But is that to report issues about the docs or the product?

Reply:
Those are for product issues and feedback. Docs feedback is with the control at the bottom of the docs page (Yes/No and follow-up verbatim feedback).